src/main/java/rs117/hd/HdPluginConfig.java

```diff
@@ -58,6 +58,18 @@ public interface HdPluginConfig extends Config
 	)
 	String generalSettings = "generalSettings";
 
+	@ConfigItem(
+			keyName = "disablePanel",
+			name = "Disable Plugin Panel",
+			description = "Disable Plugin Panel.",
+			position = 0,
+			section = generalSettings
+	)
+	default boolean disablePanel()
+	{
+		return false;
+	}
+
```

Comment:
Maybe this (and all of the logic that uses it) should be enablePanel instead of disablePanel?

Also, I don't think the panel should be on by default.

Reply:
Disagree I think it should be on by Default especially with presets ect will be there  'enablePanel ' would be better tho I agree on that

Reply:
IMO it shouldn't be on be default because of presets until presets are actually available (and you can reset the config at that point to enable it for people with it disabled if you really want to). In its current state, I don't think the panel adds enough to justify being on by default.

(I'm not sure it should be on by default even once presets are available, but that's a separate discussion)

Reply:
Like I said I disagree since it also provides info for users about the system also. But I guess we can see what the others think

Reply:
I think this panel should be on by default. It provides a lot of useful information that can be used for troubleshooting plus it's nice to have a dedicated button for the plugin to know that you have the plugin successfully installed in the first place.